packages/paste-libraries/data-visualization/src/usePasteHighchartsTheme.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {ThemeContext} from '@twilio-paste/styling-library';
+import type {ThemeShape} from '@twilio-paste/theme';
+import type Highcharts from 'highcharts';
+
+const merge = require('deepmerge');
+
+/**
+ * A hook to apply the Paste theme to Highcharts options.
+ *
+ * @param {Highcharts.Options} options - options to be provided into Highcharts for rendering.
+ * @returns {Highcharts.Options} optionsWithPasteTheme - Passed options with deepmerged Paste Theme
+ */
+export const usePasteHighchartsTheme = (options: Highcharts.Options): Highcharts.Options => {
```

Comment:
I needed to take an `options` object as argument here because the theme in Highcharts is applied over the same object that configures chart data. I was running into issues where my styles weren't applying because the options object defined different values for certain keys. So we need to take the options object, deepmerge our styles over them, and return a new object which is passed into Highcharts.  This works pretty well, as can be seen in the stories.